Storage/DatabaseStorage.php

```diff
@@ -131,11 +132,20 @@ protected function getRepository()
         return $this->getObjectManager()->getRepository($this->modelClassName);
     }
 
+    public function getClientIp(Request $request): ?string
```

Comment:
Should we add tests for this?

Reply:
this is a vendor, tests will be added on the cloud side of things